### ISSUE TITLE
v4.0.3 Segmentation fault

### ISSUE THREAD

**Issue by zchlm** [koekeishiya/yabai#1399]
```
EVENT_HANDLER_APPLICATION_LAUNCHED: OSDUIHelper (12113) is not finished launching, subscribing to finishedLaunching changes
-[workspace_context observeValueForKeyPath:ofObject:change:context:]: OSDUIHelper (12113) finished launching
EVENT_HANDLER_APPLICATION_LAUNCHED: OSDUIHelper (12113)
window_manager_create_and_add_window: ignoring AXUnknown window OSDUIHelper 1184
[1]    10839 segmentation fault  yabai --verbose
```

**Comment by koekeishiya** [comment]
Fixed in v4.0.4.

**Comment by tophee** [comment]
So this fixes #1367 ?
